build.sbt

```diff
@@ -44,7 +44,10 @@ resolvers ++= Seq(
 scalacOptions ++= Seq(
   "-feature", "-deprecation", "-unchecked",
   "-language:experimental.macros",
-  "-Ywarn-unused-import",
```

Comment:
`-Ywarn-unused-import` is no longer valid in 2.13 but `-Ywarn-unused:imports` is not valid in 2.12